service-api/src/main/java/greencity/dto/CreateAddressRequestDto.java

```diff
@@ -0,0 +1,43 @@
+package greencity.dto;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Builder
+public class CreateAddressRequestDto {
+    @Pattern(regexp = "[A-Za-zА-Яа-яЇїІіЄєҐґ ,0-9']*", message = "use only English,or Ukrainian latter")
```

Comment:
introduce 
`private static final String VALIDATION_MESSAGE = "use only English,or Ukrainian latter";` 
and use it for all fields instead of copying.